dashboard/app/models/pd/application/facilitator1819_application.rb

```diff
@@ -505,10 +509,6 @@ def destroy_fit_autoenrollment
       self.auto_assigned_fit_enrollment_id = nil
     end
 
-    def fit_workshop
-      Pd::Workshop.find(fit_workshop_id) if fit_workshop_id
-    end
-
```

Comment:
This was a [duplicate](https://github.com/code-dot-org/code-dot-org/blob/810bd1beb9505ccb539f77b29817a4f908ee3964/dashboard/app/models/pd/application/facilitator1819_application.rb#L82-L84)